forge-game/src/main/java/forge/game/card/CardFactoryUtil.java

```diff
@@ -3964,6 +3996,15 @@ else if ("Artifact".equals(t))
             String effect = "Mode$ CantBlockBy | ValidAttacker$ Creature.Self | ValidBlocker$ Creature.withoutFlying+withoutReach | Secondary$ True" +
                     " | Description$ Flying (" + inst.getReminderText() + ")";
             inst.addStaticAbility(StaticAbility.create(effect, state.getCard(), state, intrinsic));
+        } else if (keyword.startsWith("Harmonize")) {
+            String raiseEffect = "Mode$ OptionalCost | ValidCard$ Card.Self | ValidSA$ Spell.Harmonize | Secondary$ True"
+                    + " | Cost$ tapXType<1/Creature.powerEQHarmonize/creature for Harmonize> | EffectZone$ All "
+                    + " | Description$ Harmonize (" + inst.getReminderText() + ")";
+            inst.addStaticAbility(StaticAbility.create(raiseEffect, state.getCard(), state, intrinsic));
+            String reduceEffect = "Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell.Harmonize | Secondary$ True"
+                    + " | Amount$ Harmonize | EffectZone$ All | CheckSVar$ Harmonize"
```

Comment:
the problem here with extrinsic:
`addChangedCardKeywordsInternal` doesn't/can't update the host when moving to stack, so it fails to see the Announce on the old one

Reply:
yea that seems to be the place
but I'm not sure we can safely do it without breaking rollback